packages/ast-tooling/package.json

```diff
@@ -1,38 +1,38 @@
 {
-	"name": "@sveltejs/ast-tooling",
-	"private": true,
-	"version": "0.0.0",
-	"type": "module",
-	"license": "MIT",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/sveltejs/cli/tree/main/packages/ast-tooling"
-	},
-	"bugs": "https://github.com/sveltejs/cli/issues",
-	"scripts": {
-		"check": "tsc",
-		"format": "pnpm lint --write",
-		"lint": "prettier --check . --config ../../prettier.config.js --ignore-path ../../.gitignore --ignore-path .gitignore --ignore-path ../../.prettierignore"
-	},
-	"files": [
-		"dist"
-	],
-	"exports": {
-		".": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
-		}
-	},
-	"devDependencies": {
-		"@babel/parser": "^7.24.7",
-		"ast-types": "^0.14.2",
-		"dom-serializer": "^2.0.0",
-		"domhandler": "^5.0.3",
-		"domutils": "^3.1.0",
-		"htmlparser2": "^9.1.0",
-		"postcss": "^8.4.38",
-		"recast": "^0.23.7",
-		"silver-fleece": "^1.2.1",
-		"zimmerframe": "^1.1.2"
-	}
+  "name": "@sveltejs/ast-tooling",
```

Comment:
can you revert the spacing changes in this file?

Reply:
not sure since it seems specific to your setup and I don't know what editor you use, etc. we do have prettier setup in this repo, which I think should format things appropriately